docs/pages/versions/unversioned/sdk/app-loading.md

```diff
@@ -56,7 +56,7 @@ export default class App extends React.Component {
   async _cacheResourcesAsync() {
     const images = [require('./assets/snack-icon.png')];
 
-    /* @info Read more about <a href='../guides/preloading-and-caching-assets.html'>Preloading and Caching Assets</a> */
+    /* @info Read more about <a href='../archive/classic-updates/preloading-and-caching-assets.html'>Preloading and Caching Assets</a> */
```

Comment:
```suggestion
    /* @info Read more about <a href='/archive/classic-updates/preloading-and-caching-assets.html'>Preloading and Caching Assets</a> */
```

I tested these changes locally and the redirect was not working until I removed the `..`. Here is a short demo of that:

![CleanShot 2022-08-02 at 02 56 44](https://user-images.githubusercontent.com/10234615/182249877-824148fd-fa4a-44f4-b0e6-0868b50b270e.gif)